test/postman/inference_api_test_collection.json

```diff
@@ -91,6 +91,8 @@
 							"        pm.expect(validators[validator](actual_obj, expected_obj)).to.be.true;",
 							"    });",
 							"",
+							"} else if (type_response === \"binary\") {",
```

Comment:
Should we write the expected response to a resource file and then compare the binary contents instead of ignoring the response completely?

Reply:
responded below.